### PR TITLE
Remove ModuleAdminControllers from SEO & URLs page

### PR DESCRIPTION
**Pull request by Matt75**
| Questions     | Answers
| ------------- | -------------------------------------------------------
| Description?  | `Module::$controllers` is an undocumented reserved var used to register `ModuleFrontController` in SEO & URLs page on BO in order to allow merchant to customize rewrite link and page title, meta keyword, meta description... It should never be used for a `ModuleAdminController`
| Type?         | bug fix
| BC breaks?    | no
| Deprecations? | no
| Fixed ticket? | 
| How to test?  | See below

## How to test ?

Install current module, go to BO > Preferences > SEO & URLs, search `ps_faviconnotificationbo`. and see it in the list: this list item is bad.
With new module
- if you upgrade, the bad list item is gone
- if you install, it will not add the bad list item

ps_faviconnotificationbo-admincontrollers
ps_faviconnotificationbo-admincontroller

### CODE REVIEW

File: upgrade/upgrade-2.0.1.php [hunk 1]
```diff
@@ -0,0 +1,61 @@
+<?php
+/**
+ * 2007-2016 PrestaShop
```
Comment (matks) [inline upgrade/upgrade-2.0.1.php:3]: As it's a new file, could we use up-to-date license header please ? 😉

**Comment by matks** [comment]
@Matt75 I improved the "how to test" section please tell me if I did a mistake

**Comment by Matt75** [comment]
Note for QA, upgrade script will be launch only if you increment the version of the module.
https://github.com/PrestaShop/ps_faviconnotificationbo/blob/dev/ps_faviconnotificationbo.php#L45
`$this->version = '2.0.1';`

**Comment by Progi1984** [comment]
Thanks @Matt75

**Pull request merged**